tony-core/build.gradle

```diff
@@ -1,3 +1,6 @@
+plugins {
+  id "com.commercehub.gradle.plugin.avro" version "0.9.0"
```

Comment:
Just by applying this plugin, do the Avro classes automatically get generated as part of build?

Reply:
Yes.